src/scripts/runscripts.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+HELP=false #Display help
+QUIET=false #Quiet mode
+COUNTER=0 # The count of tests run
+FAIL=0 # The count of failed tests
+PASS=0 # The count of passed tests
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    --help)
+    HELP=true
+    shift
+    ;;
+    -q)
+    QUIET=true
+    shift
+    ;;
+    --test-folder)
+    TEST_FOLDER="$2"
+    shift
+    shift
+    ;;
+    --jerry-location)
+    JERRY_LOCATION="$2"
+    shift
+    ;;
+    *)
+    shift
+    ;;
+esac
+done
+if [ $HELP = true ] ; then
+	echo "Usage: runscripts.sh [OPTION]"
```

Comment:
In `bash` it is better to something like: `echo "Usage: $(basename $0) [OPTION]"`